strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/repository/proxied/ProxyRepositoryArtifactResolver.java

```diff
@@ -77,7 +77,17 @@ public InputStream getInputStream(final String storageId,
                                                                                          remoteRepository.getPassword()))
         {
             final LayoutProvider layoutProvider = layoutProviderRegistry.getProvider(repository.getLayout());
-            final URI resource = layoutProvider.resolveResource(repository, path);
+
+            URI resource;
+            try
+            {
+                resource = layoutProvider.resolveResource(repository, path);
+            }
+            catch(IllegalArgumentException e)
+            {
+                getLogger().debug("Cannot resolve invalid path " + path);
```

Comment:
Could you please add caught Exception `e` as a last argument of `debug` method ? This way we will not loose the stacktrace.